EventVisualisation/Base/include/EventVisualisationBase/EventManager.h

```diff
@@ -45,30 +47,31 @@ class EventManager : public TEveEventManager, public TQObject
       Digits,   ///< Digits
       Clusters, ///< Reconstructed clusters (RecPoints)
       ESD,      ///< Event Summary Data
-      AOD       ///< Analysis Object Data
+      AOD,      ///< Analysis Object Data
+      NoData,   ///< no data was loaded
+      NdataTypes///< number of supported data types
```

Comment:
what about moving the whole definition to the new `VisualizationConstants.h` file? And then even the return strings for `EventManager::getDataTypeName` could be defined there, allowing for return of `const std::string&`.

Reply:
Good point @matthiasrichter, it should be in the constants. I moved it there.